### PR TITLE
feat: add Message Handling for  jitsi/lib-jitsi-meet/pull/2339

### PR DESCRIPTION
**Pull request by Bloodiko**
This is the corresponding xmpp prosody plugin addition in the breakout_rooms plugin for the PR https://github.com/jitsi/lib-jitsi-meet/pull/2339

There may be changes necessary to this, as this lua code is not really complying with the TS Objects types. I was unable to figure out how the objects are correctly typed, so I just gathered as much info as I could from other plugins. 
(lua is definitely not my main language, but it was requested in the other PR that this PR should also be created thus, here it is.) 

TLDR: Goal is to allow moderators to send single text messages or a whole broadcast text message to all rooms within the same conference. 

The "to" argument was chosen to be a composite of source room and occupant/participant nick, to give visual clarity that the message came from a different room. 
This can be adjusted if needed.

### CODE REVIEW

**Review by saghul — DISMISSED** [review]
LGTM! @damencho can you PTAL when you get a chance?

**Comment by Bloodiko** [comment]
Wait a second, the Lua check failed with an unterminated String, I'll check and fix later today. 
Didn't knew there is a Lua pipeline.
